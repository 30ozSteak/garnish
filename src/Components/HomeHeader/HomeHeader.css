.home-header {
  padding: 1rem;
  color: var(--NotBlack);
  width: 80%;
}

.homeheader h1 {
  font-weight: 400;
  font-size: 1.9rem;
}

.home-header h3 {
  font-size: 1rem;
  font-weight: 500;
}
