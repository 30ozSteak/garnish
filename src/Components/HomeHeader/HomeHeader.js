import React from "react";
import "./HomeHeader.css";

const HomeHeader = () => {
  return (
    <div className="home-header">
      <h1>Hello, nick</h1>
      <h3>Here's the least amount of news for you to get through the day</h3>
    </div>
  );
};

export default HomeHeader;
