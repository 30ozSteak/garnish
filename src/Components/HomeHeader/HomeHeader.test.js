import React from "react";
import HomeHeader from "./HomeHeader";
import { shallow } from "enzyme";

describe("HomeHeader", () => {
  it("should resemble the snapshot", () => {
    const wrapper = shallow(<HomeHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
