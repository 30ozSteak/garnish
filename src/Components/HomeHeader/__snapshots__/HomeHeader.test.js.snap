// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeHeader should resemble the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <HomeHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Hello, nick
        </h1>,
        <h3>
          Here's the least amount of news for you to get through the day
        </h3>,
      ],
      "className": "home-header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Hello, nick",
        },
        "ref": null,
        "rendered": "Hello, nick",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Here's the least amount of news for you to get through the day",
        },
        "ref": null,
        "rendered": "Here's the least amount of news for you to get through the day",
        "type": "h3",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Hello, nick
          </h1>,
          <h3>
            Here's the least amount of news for you to get through the day
          </h3>,
        ],
        "className": "home-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Hello, nick",
          },
          "ref": null,
          "rendered": "Hello, nick",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Here's the least amount of news for you to get through the day",
          },
          "ref": null,
          "rendered": "Here's the least amount of news for you to get through the day",
          "type": "h3",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
